docker/nitro/anthony.toml

```diff
@@ -11,7 +11,7 @@ pk = ""
 # Chain Public Address: 0x56032FA02b2592338b9A42B1865d5a5BEEaF2B75
 chainpk = ""
 
-naaddress = "0xe32d4B5C5a80660710f6a2aD3cB1c11664138F34"
+naaddress = "0xe1790ea824035184a3bf344e087fb61744992545"
 vpaaddress = "0x4D1a804e1cE383D75032C52878553212992C98D7"
```

Comment:
Not sure if these are even used but new addresses for the other two contracts are:
```
* ConsensusApp: 
    * txHash: 0xffd6975b128363acc953b7dd5afc3590b1a3dbd5dad818b79d614ba4fb027550
    * address: 0x1e12b91f71D6fAEA0bcE97e739253A1AC44E5CFf
* VirtualPaymentApp 
    * txHash: 0xf9a0de4b60dddb19adb3e8d86a32e297ce40cee992017f7240147bcf8a4d74b6
    * address: 0x95EfacCb38106C249F5ddC25b71677d5aF6d31A0
```